dandi/cli/cmd_download.py

```diff
@@ -86,7 +86,12 @@ def get_metavar(self, param):
 @click.option(
     "--sync", is_flag=True, help="Delete local assets that do not exist on the server"
 )
-@instance_option(default=None)
+@instance_option(
+    default=None,
+    help="Instance of DANDI archive to use. If URLs provided, we would ensure that "
+    "they point to the same instance as specified. If no URLs provided, that "
+    "instance will be used for identifier found in local dandiset.yaml.",
```

Comment:
Suggested rewording:

> DANDI Archive instance to download from.  If any URLs are provided, they must point to the given instance.  If no URL is provided, and there is a local dandiset.yaml file, the Dandiset with the identifier given in the file will be downloaded from the given instance.

Reply:
could you please send it as suggestion (select multiple lines before clicking on suggestion icon)

Reply:
indeed is better than mine, accepted